PATCH RUN @ 2026-09-26T02:12:13.0463947Z
patch 1 = patch1-csharp-compile-verdict-5f3e7fb441.diff (5303 B, sha256 6b0ee95c2abde614a4ec0ed2e6e4b7eb00ee39a75ebddf3906737d43cda02353)
  --check exit=0 out=
  apply   exit=0 out=
patch 2 = patch2-projectsettings-section-publish-96f631addb.diff (23683 B, sha256 70decff99a232fd5866f37cd8bd952ffa40afee8801d2bfc1eaa4b5be46b4c4c)
  --check exit=0 out=
  apply   exit=0 out=
patch 3 = patch3-save-preserving-text-2f85141a74.diff (20309 B, sha256 41867b2962fce8c764da13fe9812c8c67a2bba22c16be5923c0c53206a5fae7e)
  --check exit=0 out=
  apply   exit=0 out=
--- post-patch sha256 ---
core/config/project_settings.cpp | 100000 B | f5e6304bf48f420087cd53ef4731783e74cea8d757068a81bc3dda4bee2c2b5e | DIFF
core/config/project_settings.h | 18626 B | c41ed5e47f7835b5aeb097ab05efced56719f432211489409eea4d50d228dc2e | DIFF
editor/editor_node.cpp | 390377 B | 85d2ed53d40345873c68266f8050d42749239610c9833f6ab43fbd3fc2e23977 | DIFF
modules/mono/csharp_script.cpp | 86677 B | 0897a8383461f016df417878ac21ac7dfa67c13bcf5eff0b9ae5353a60b45539 | DIFF
modules/mono/csharp_script.h | 22089 B | e96ac9ae5e7b5a938cf034404ac5fc2234b60c50eccfb03069d449d3648fc792 | DIFF
